### ISSUE TITLE
[Feature Request] keep custom date range

### ISSUE THREAD

**Issue by flighter18** [Yooooomi/your_spotify#99]
Type: Feature Request
Description: i would like to keep custom date range while browsing the websites.
For example: if you choose a date range and change from the "top songs" page to "top artists" page the date range will be still present and you don't have to rechoose it again.

**Comment by Yooooomi** [comment]
Yes! This is something that could be very valuable to add, to smooth the browsing of stats.

**Comment by Yooooomi** [comment]
It is now implemented, if you could try it out to see if it fit your needs it would be great! Feel free to close this issue if this is what you wanted.

**Comment by flighter18** [comment]
It works nicely, thank you.

But i got one more point :)
maybe it would be great if you see the choose timerange while browser.
You cold implement it at the right corner under "IntervalSelector_radiogroup__oepEJ"
